silkworm/silkrpc/core/evm_trace.cpp

```diff
@@ -1514,4 +1549,57 @@ void CreateTracer::on_execution_start(evmc_revision, const evmc_message& msg, ev
                << ", code: " << silkworm::to_hex(code);
 }
 
+void EntryTracer::on_execution_start(evmc_revision, const evmc_message& msg, evmone::bytes_view code) noexcept {
+    auto sender = evmc::address{msg.sender};
+    auto recipient = evmc::address{msg.recipient};
+    auto code_address = evmc::address{msg.code_address};
+    bool create = (!initial_ibs_.exists(recipient) && recipient != code_address);
+    auto input = silkworm::ByteView{msg.input_data, msg.input_size};
+
+    auto str_value = "0x" + intx::hex(intx::be::load<intx::uint256>(msg.value));
+    auto str_input = "0x" + silkworm::to_hex(input);
+    if (str_input == "0x") {
+        str_input = "0x" + silkworm::to_hex(code);
+    }
+
+    if (create) {
+        result_.entries.push_back(TraceEntry{"CALL", msg.depth, sender, recipient, str_value, str_input});
+    } else {
+        bool in_static_mode = (msg.flags & evmc_flags::EVMC_STATIC) != 0;
+        switch (msg.kind) {
+            case evmc_call_kind::EVMC_CALL:
+                in_static_mode ? result_.entries.push_back(TraceEntry{"STATICCALL", msg.depth, sender, recipient, "", str_input}) : result_.entries.push_back(TraceEntry{"CALL", msg.depth, sender, recipient, str_value, str_input});
+                break;
+            case evmc_call_kind::EVMC_DELEGATECALL:
+                result_.entries.push_back(TraceEntry{"DELEGATECALL", msg.depth, recipient, code_address, "", str_input});
+                break;
+            case evmc_call_kind::EVMC_CALLCODE:
+                result_.entries.push_back(TraceEntry{"CALLCODE", msg.depth, sender, recipient, str_value, str_input});
+                break;
+            case evmc_call_kind::EVMC_CREATE:
+            case evmc_call_kind::EVMC_CREATE2:
+                break;
+        }
+    }
+
+    SILK_DEBUG << "EntryTracer::on_execution_start: gas: " << std::dec << msg.gas
+               << " create: " << create
+               << ", msg.depth: " << msg.depth
+               << ", msg.kind: " << msg.kind
+               << ", sender: " << sender
+               << ", recipient: " << recipient << " (created: " << create << ")"
+               << ", code_address: " << code_address
+               << ", msg.value: " << intx::hex(intx::be::load<intx::uint256>(msg.value))
+               << ", code: " << silkworm::to_hex(code)
+               << ", msg.input_data: " << to_hex(ByteView{msg.input_data, msg.input_size})
+               << "\n";
```

Comment:
Differently from old `SILKRPC_` macros, there is no need to append `\n` with `SILK_` macros because it's already done in the logging facility.